File: FrontEnd/AnguarTask/app/src/app/Models/order.ts
import { OrderItem } from "./order-item"
import { Receipt } from "./receipt"
import { User } from "./user"

export interface Order {
    id:number,
    createdAt:Date,
    updatedAt:Date,
    status:string,
    totalAmount:number,
    customer:User,
    customerId:number,
    orderItems:OrderItem[],
    receipt:Receipt
}

File: BackEnd/StoreWebApi/StoreDomain/Models/Order.cs
namespace StoreDomain.Models
{
    public class Order
    {
        public int Id { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        public string Status { get; set; }
        public int TotalAmount { get; set; }
        public User Customer { get; set; }
        public int CustomerId { get; set; }
        public List<OrderItem> OrderItems { get; set; }
        public Receipt Receipt { get; set; }
        //public int ReceiptId { get; set; }
    }
}
